newrelic-integration-e2e/internal/docker/docker.go

```diff
@@ -1,17 +1,44 @@
 package docker
 
 import (
+	"fmt"
 	"os"
 	"os/exec"
+	"strings"
 )
 
 const (
 	dockerComposeBin = "docker-compose"
 )
 
-func DockerComposeUp(path string) error{
-	cmd := exec.Command(dockerComposeBin, "-f", path, "up", "-d")
+
+func DockerComposeRun(path string,container string, envVars map[string]string) error {
+	DockerComposeBuild(path,container)
+	args := []string{"-f", path,"run"}
```

Comment:
why run and not up? any specific reason?

Reply:
`docker-compose up` does not support `-e` flag.  So in case of using the `docker-compose up` we will need to create a file with the license key and passing with the flag `--env_file` or add the license key in the docker-compose (in this case we would need to create the docker-compose dynamically)